### PR TITLE
Simplify the process of ImportSST::Upload API

### PR DESCRIPTION
**Pull request by huachaohuang**
Process upload stream without token and locking.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -82,7 +82,8 @@ git = "https://github.com/pingcap/murmur3.git"
 git = "https://github.com/pingcap/rust-rocksdb.git"
 
 [dependencies.kvproto]
-git = "https://github.com/pingcap/kvproto.git"
+git = "https://github.com/huachaohuang/kvproto.git"
```
Comment (Hoverbear) [inline Cargo.toml]: Before merge will we go back to using the `pingcap` repository?

Comment (huachaohuang) [reply Cargo.toml]: Yes, after I have two LGTMs.

**Comment by siddontang** [comment]
Much better

LGTM

**Comment by breeswish** [comment]
LGTM

**Pull request merged**